docker-compose.yml

```diff
@@ -8,38 +8,16 @@ secrets:
     file: ./genfiles/monitor_sign-key.pem
 
 services:
-  # Only works when when replicas = 1 for reach monitored service.
-  # Kubernetes sidecars are needed for replicas > 1. 
-  prometheus-to-sd:
-    container_name: prometheus-to-sd
-    image: gcr.io/google-containers/prometheus-to-sd:v0.2.6
-    entrypoint:
-      - /monitor
-      - --stackdriver-prefix=custom.googleapis.com
-      - --source=logserver:http://log-server:8091/metrics
-      - --source=logsigner:http://log-signer:8091/metrics
-      - --source=mapserver:http://map-server:8091/metrics
-      - --source=keyserver:http://server:8081/metrics
-      - --source=sequencer:http://sequencer:8081/metrics
-      - --pod-id=prometheus-to-sd
-      - --namespace-id=default
-      - --metrics-resolution=5s
-
   prometheus:
     depends_on:
       - server
       - sequencer
     image: gcr.io/key-transparency/prometheus:${TRAVIS_COMMIT}
-    build:
-      context: .
-      dockerfile: deploy/prometheus/Dockerfile
-    restart: always
     ports:
       - 9090:9090
 
   db:
     image: gcr.io/trillian-opensource-ci/db_server:latest
```

Comment:
Everything should really be `${TRAVIS_COMMIT}`. The behavior of `latest` is non-deterministic and can cause unexpected behavior.

For Trillian, at the moment, I'm happy to track their latest images because I have integration testing. I'm not sure what the best practice for this is. Pinning to an old version is less than perfect because I don't know what the trillian docker durability guarantees are

Reply:
Agreed, I'm also _not_ a fan of `latest.

We can discuss with Trillian and figure out whether there's a mechanism to safely pin to an image tag. Would you mind adding a TODO() to denote the desire to move away from `latest`? Thanks.